src/sentry/static/sentry/app/views/settings/components/panelAlert.jsx

```diff
@@ -7,10 +7,21 @@ import Alert from '../../../components/alert';
 
 // Margin bottom should probably be a different prop
 const StyledPanelAlert = styled(Alert)`
-  margin: ${p => `-${p.m * 2 + 1}px -${p.m * 2 + 1}px ${p.mb * 3}px`};
+  ${p =>
+    typeof p.m !== 'undefined'
+      ? `margin: -${p.m * p.theme.grid + 1}px -${p.m * p.theme.grid + 1}px;`
```

Comment:
: O

Reply:
is there a better place to have this logic, or maybe could you write it with booleans instead of short circuiting or something?
 
it's shippable, but it's a bit gnar

Reply:
Agreed, I made similar comments in my review when this component got added. Let's make a followup PR for this.